src/ChannelServer/Util/GmCommands.cs

```diff
@@ -260,11 +260,12 @@ private CommandResult HandleWarp(ChannelConnection conn, Character character, Ch
 
 		private CommandResult HandleItem(ChannelConnection conn, Character character, Character target, string command, string[] args)
 		{
-			if (args.Length < 2)
+			if (args.Length < 3)
```

Comment:
Why not make the amount parameter optional? That way you don't need it every time you want to spawn something, especially for things like equipment, that will always be 1.